saleor/checkout/utils.py

```diff
@@ -603,25 +604,34 @@ def get_valid_shipping_methods_for_checkout(
     )
 
 
-def is_valid_shipping_method(checkout_info: "CheckoutInfo"):
-    """Check if shipping method is valid and remove (if not)."""
-    if not checkout_info.shipping_method:
-        return False
+def get_valid_collection_points_for_checkout(
+    lines: Iterable["CheckoutLineInfo"],
+    checkout_info: "CheckoutInfo",
+    country_code: Optional[str] = None,
+    quantity_check: bool = True,
```

Comment:
Could we add a docstring for this function? `quantity_check` requires explanation as it's not obvious why or when you should use it.

Reply:
Docstrings has been added.